protocol/x/clob/e2e/equity_tier_limit_test.go

```diff
@@ -726,7 +726,7 @@ func TestPlaceOrder_EquityTierLimit_OrderExpiry(t *testing.T) {
 				resp := tApp.CheckTx(tx)
 				if tc.expectError {
 					require.Conditionf(t, resp.IsErr, "Expected CheckTx to error. Response: %+v", resp)
-					require.Contains(t, resp.Log, "Opening order would exceed equity tier limit of 1.")
+					require.Contains(t, resp.Log, "Opening order would exceed equity tier limit of 1. Order count: 1,")
```

Comment:
Do we have any E2E tests that verify equity tier limits that are greater than 1?

Reply:
Added cases with an equity tier count of 2.